### PR TITLE
bears: Strings are not pre-splitted

### PR DESCRIPTION
**Pull request by Ana06**
Strings in method verify_local_bear() were not pre-splitted.

Closes https://github.com/coala-analyzer/coala-bears/issues/322

I open another PR because I broke https://github.com/coala-analyzer/coala-bears/pull/548 :persevere:

### CODE REVIEW

**Comment by Makman2** [comment]
again sorry for the last PR ... I forget sometimes on which repository I'm/someone is working on :3
but what's not killing us makes us stronger :+1: :P

alright, there was a completely new bear + test added, `CSecurityBear`. If you adapt these test, then everything should work nicely :)

**Comment by Makman2** [comment]
and then we can merge :+1: :grapes:

File: tests/xml2/XMLBearTest.py [hunk 1]
```diff
@@ -28,30 +35,15 @@
...
-    "test_files",
-    "concept-invalid.xml"), 'r'))
+valid_xml_path = load_testdata("note.xml")
+
+valid_xml_url = load_testdata("concept-valid.xml")
+
+invalid_xml_schema = load_testdata("xsd-error.xml")
+
+invalid_xml_dtd = load_testdata("dtd-error.xml")
+
```
Comment (Makman2) [inline tests/xml2/XMLBearTest.py]: I think you can remove these newlines, so we are saving another 5 lines making this commit looking even better :D

**Comment by Ana06** [comment]
I think it can be merged now! :tada:

**Comment by Asnelchristian** [comment]
This awesome :) 
Good job :) :+1:

**Comment by sils1297** [comment]
houuuuuuh awesome ! :) @Ana06 are you at our gitter channel btw.?

**Comment by Asnelchristian** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/582/commits/2f6100d45abd89f2a24ab0e56ccc533a1089b97a

**Comment by Asnelchristian** [comment]
i am merging now :)

**Comment by Asnelchristian** [comment]
@rultor merge

**Comment by sils1297** [comment]
Oh yes! I've been looking forward to that - @Ana06 you're awesome!

**Comment by Ana06** [comment]
haha :wink:  @sils1297 No, I am not at your Gitter chanel. What channel is it?

**Comment by sils1297** [comment]
https://gitter.im/coala-analyzer/coala

**Pull request merged**